Bw assurance research path page add. (#150)
nhsengland/datascience: docs/our_work/ds278_Assurance_Research_Path.md

Reviewer: looks good overall! just a couple of minor changes, main thing that needs changing ebfore I can approve is the tags and then also it needs adding to the navigation. currently when opening the website with this it appears outside of any navigation, so it needs to be placed in the correct places in this file: https://github.com/nhsengland/datascience/blob/main/mkdocs.yml

---

> title: 'DS_278: Assurance Research Path'

Reviewer: I wouldnt give the name of it the DS code, cause this is public facing and that doeasnt mean anything to anyone oputside the team + just clutters up the titles

summary: 'Investigating new techniques to test and assure AI and machine learning solutions in the NHS'
origin: 'NHS England'

> tags: ['NLP','LLM','GENAI',Assurance, Machine Learning, Classification, Computer Vision, Work in Progress]

Reviewer: please check this page for the currently existing tags https://nhsengland.github.io/datascience/our_work/tags/ , and try and align to them, if you dont think theyre appropriate, let me know any suggestions you might have!

---

> The NHS AI Quality Community of Practice (AIQ CoP) is a network of testing and assurance professionals dedicated to supporting the assurance of healthcare AI. While not an assurance function itself, the AIQ CoP provides guidance, resources, and advice to assist colleagues and organisations in navigating the complexities of AI assurance. While the project sits within NHS England, the remit of the AIQCOP is the whole of the NHS. 

Reviewer: be consisrent on how you capitalise the acronym so either AIQ CoP or AIQCOP every time

The AIQCOP conducts cutting-edge research to address real-world challenges in AI assurance. Current research areas include boundary analysis in image classifiers, focusing on understanding and validating how AI models make classification decisions; bias in AI training data, investigating scenario and demographic biases to mitigate risks in model performance; literacy bias in chatbots and AI search systems, aiming to ensure equitable access for users with varying literacy levels; and exploring AI-driven automation of risk logs, test scripts, and test data generation. This work combines traditional assurance techniques with novel approaches to enhance the reliability and fairness of AI systems.

For years now the AIQCOP has relied on embedded Data Scientists from NHS England to help advise on the technical aspects of AI Assurance. 

> Read our Mixup Assurance paper here:  - [Link to paper](https://arxiv.org/abs/2311.13978)

Reviewer: ```suggestion
Read our Mixup Assurance paper here [on ArXiv](https://arxiv.org/abs/2311.13978)
```

just think it looks nicer this way! might also be worth giving a very brief explanation of what Mixup is

> RISE is an experimental tool which can help assurers, data scientists and clinicians to evaluate Ai Classifiers, read more here: - [Link to blog ](https://nhsengland.github.io/datascience/articles/2024/12/12/RISE_tool/)  

Reviewer: ```suggestion
RISE is an experimental tool which can help assurers, data scientists and clinicians to evaluate Ai Classifiers, read more [on the blog](https://nhsengland.github.io/datascience/articles/2024/12/12/RISE_tool/)  
```

![An image of Will and Ben at the Health and Analytics Conference talking about AI Assurance](../images/will_and_ben_assurance.jpg)
*Will and Ben at the Health and Care Analytics Conference talking about AI Assurance*

Most information about the work in the Assurance Research Path appears on the AIQCOP sharepoint site, access may need to be requested to view.

[AI Quality Community of Practice](https://nhs.sharepoint.com/sites/X26_AssuranceCOE/SitePages/AI-&-Quality-Assurance.aspx?csf=1&web=1&e=HTahJY)

> Alternatively you can always email us at: nhsdigital.instant@nhs.net

Reviewer: make this a clickable email address

> 

Reviewer: might want to include a little table at tghe bottom with a sumarry of all the links shared (for consistency with other projects) like we have here: https://nhsengland.github.io/datascience/our_work/ds218_rap_community_of_practice/ 

see here: https://github.com/nhsengland/datascience/blob/main/docs/our_work/a_and_e_forecasting_tool.md?plain=1 how to code it in

Author: I think because the links are towards the bottom of the page anyway I won't do this.
